main.cpp

```diff
@@ -119,288 +154,357 @@ struct JumpStarter
-    //2)heat a room
-    float heatRoom( float initialTemperature = 55 );
-    //3)select the fan speed
+    double removeHeat( int roomHeatAmount = 8000 );      //changed float to doble to avoid warnings
+    double heatRoom( int initialTemperature = 55 );
     int selectFanSpeed( int selectedSpeed );
 };
-/*
-5)USB interface
-5 properties:
-    1)speed in Mb per second
-    2)standard (1.0 or 2.0 for USB ) 
-    3)cable length 
-    4)connector type
-    5)current it can receive for powering the board
-3 things it can do:
-    1)get the program to be stored in memory
-    2)report errors
-    3)transmit serial data back to computer
- */
+
+double AirConditioningUnit::removeHeat( int roomHeatAmount ) 
+{
+    return roomHeatAmount - coolingBtu*.75;
+}
+
+double AirConditioningUnit::heatRoom ( int initialTemperature )
+{
+    return initialTemperature + heatherPower*0.0005;
+}
+
+
+///////////////////////////////////////////////////////////////////////
 struct UsbInterface
 {
-    //1)speed in Mb per second
     int speed = 40;
-    //2)standard (1.0 or 2.0 for USB ) 
     int standard = 1;
-    //3)cable length
     float length = 5; 
-    //4)connector type
     char connectorType = 'A';
-    //5)current it can receive for powering the board
     float currentDrawForPower = 50;
 
-    //1)get the program to be stored in memory
     int openLinkToGetData();
-    //2)report errors
     char sendErrorToComputer( int errorCode = 0 );
-    //3)transmit serial data back to computer
-    int sendDataStreamToComputer ( int streamLength = 10 );
+    int sendDataStreamToComputer( int streamLength = 10 );
 };
-/*
-6)DA converters
-5 properties:
-    1)amount of channels
-    2)bit depth
-    3)clocking speed
-    4)max output current
-    5)output impedance
-3 things it can do:
-    1)provide a constant voltaje for controlling a device
-    2)drive a transistor
-    3)read a constant sensor voltage 
- */
+
+int UsbInterface::openLinkToGetData()
+{
+    return 1;
+}
+
+char sendErrorToComputer( int errorCode )
+{
+    if( errorCode == 0 )
+    {
+    return '0';
+    }
+    else return 'f';
+}
+
+int sendDataStreamToComputer( int streamLength )
+{
+    return streamLength;
+}
+///////////////////////////////////////////////////////////////////////
 struct Converter
 {
-    //1)amount of channels
     int channels = 5;
-    //2)bit depth
     int bitDepth =16;
-    //3)clocking speed
     int clockSpeed = 1;
-    //4)max output current
     float maxCurrent = 10;
-    //5)output impedance
     char adOrDa = 'a';
 
-    //1)provide a constant voltaje for controlling a device
     void setVoltage( int vLevel = 25, int channel = 1 );
-    //2)drive a transistor
     void setBoolOut( bool status, int channel = 2 );
-     //3)read a constant sensor voltage
     int readConstantValue( int channel = 3 );
+};
 
+void Converter::setVoltage( int vLevel, int channel )
+{
+    int dummy = 0;
+    
+    if ( adOrDa == 'd' )
+    {
```

Comment:
be consistent with your spacing of `( )`.  Look at the Readme.md coding standard.
- [ ] revise